core/controllers/question_editor.py

```diff
@@ -30,11 +30,19 @@ class QuestionCreationHandler(base.BaseHandler):
     """A handler that creates the question model given a question dict."""
 
     @acl_decorators.can_manage_question_skill_status
-    def post(self, skill_id):
+    def post(self, skill_ids):
         """Handles POST requests."""
-        skill_domain.Skill.require_valid_skill_id(skill_id)
-        skill = skill_services.get_skill_by_id(skill_id, strict=False)
-        if skill is None:
+        skill_ids_list = skill_ids.split(',')
```

Comment:
skill_ids = comma_separated_skill_ids.split(',')

(In general, the normal representation should have the simplest name. The encoded one should have a longer name because it's more of a "special case".)

Reply:
Done.